tests/main/snap-confine-from-core/task.yaml

```diff
@@ -24,6 +24,15 @@ execute: |
 
     echo "Ensure we re-exec by default"
     snap list
+
```

Comment:
I think this way is simpler and widly used on the tests:
for _ in $(seq 10); do
        if get_journalctl_log | MATCH "DEBUG: restarting into" ; then
            break
        fi
        sleep 1
    done

Reply:
Done.